deps.qt/qt6.ps1

```diff
@@ -82,14 +86,19 @@ function Configure {
         $QtBuildConfiguration = '-release'
     }
 
-    $BuildCommand = "..\..\..\qt6\configure -opensource -confirm-license ${QtBuildConfiguration} -nomake examples -nomake tests -schannel -no-dbus -no-freetype -no-icu -no-openssl -no-feature-androiddeployqt -no-feature-pdf -no-feature-printsupport -no-feature-qmake -no-feature-sql -no-feature-testlib -no-feature-windeployqt -DQT_NO_PDF -prefix ${BuildPath}"
+    $BuildCommand = "..\..\..\qt6\configure " 
+
+    if (${Target} -eq "ARM64") {
+        $BuildCommand += "-platform win32-msvc -xplatform win32-arm64-msvc -qt-host-path '$($Env:Qt6_DIR)' "
+    }
+    
+    $BuildCommand += "-opensource -confirm-license ${QtBuildConfiguration} -nomake examples -nomake tests -schannel -no-dbus -no-freetype -no-icu -no-openssl -no-feature-androiddeployqt -no-feature-pdf -no-feature-printsupport -no-feature-qmake -no-feature-sql -no-feature-testlib -no-feature-windeployqt -DQT_NO_PDF -prefix ${BuildPath}"
 
     $Params = @{
         BasePath = (Get-Location | Convert-Path)
         BuildPath = "."
         BuildCommand = "${BuildCommand}"
         Target = $Target
-        HostArchitecture = $Target
```

Comment:
This is necessary to build x86 artifacts on x64 hosts, do not remove.

Reply:
disabled